### DISCUSSION TITLE
Bookmarklet Exec don't work

### DISCUSSION THREAD

**Discussion by AnOrangeCone** [3kh0/ext-remover#1018] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
[Screen recording 2024-04-19 09.05.26.webm](https://github.com/3kh0/ext-remover/assets/167538473/d6be54f1-1f0a-4c92-9d97-466cc1fda2fc)

**Comment by Blobby-Boi** [comment, marked as answer]
the servers may just be down or smth
also I think very few people if not just one person can use it at a time

**Reply by AshtonDavies** [reply]
> the servers may just be down or smth
> also I think very few people if not just one person can use it at a time

Whoever links their IP disconnects the previous person, and that other person is left with their cached DNS.

**Reply by Blobby-Boi** [reply]
oh ok

**Reply by AnOrangeCone** [reply]
Do you know what's wrong? Is the link broken or am I doing something wrong.
I'm on v120 if that helps.

**Reply by Blobby-Boi** [reply]
Idk sometimes the server is just not working... You can probably just use a javascript executor website instead cuz its basically the same thing since the sidepanel feature is not done. Or if you want to be able to run bookmarklets on any page use like ublock or snap&read

**Reply by AshtonDavies** [reply]
He did it wrong though.

**Reply by Blobby-Boi** [reply]
Oh, I didn’t realize

**Comment by AshtonDavies** [comment]
You linked with the second server for unblocking.

**Reply by CGRmags** [reply]
@AshtonDavies I though he did it right what did he do wrong?

**Reply by AshtonDavies** [reply]
He was supposed to click the second link rather than the last.

**Reply by AnOrangeCone** [reply]
the reason i used it is because I have lightspeed but that doesnt work and when i use the correct one it just takes me to the chrome web store i even tried it on my test chromebook at home
